### PR TITLE
release-notes: draft v0.94.4 release notes

### PR DESCRIPTION
**Pull request by dachary**
Signed-off-by: Loic Dachary <ldachary@redhat.com>

### CODE REVIEW

**Comment by dachary** [comment]
@liewegas do you think [hammer with these notable changes](https://github.com/dachary/ceph/blob/ba6c6898dbb2b289fd682c47ac351193afca799a/doc/release-notes.rst#notable-changes-4) should be published as v0.94.4 ?

**Comment by dachary** [comment]
@liewegas ping ? The v0.94.3 release notes referenced issues from the Backport tracker, which is confusing to most readers. These release notes reference issues from the Bug/Feature/Fix tracker instead (@theanalyst updated the script to look them up from the Backport issue).

**Comment by liewegas** [comment]
I've been waiting to m ake sure that we don't need anything else in 0.94.4 to facilitate infernalis upgrades, but it looks like we're in good shape.

File: doc/release-notes.rst [hunk 1]
```diff
@@ -508,6 +508,149 @@ Notable Changes
 * rocksdb: update to latest (Xiaoxi Chen)
 * rpm: loosen ceph-test dependencies (Ken Dreyer)
 
+v0.94.4 Hammer (draft)
+======================
+
+This Hammer point release ...
```
Comment (liewegas) [inline doc/release-notes.rst]: This Hammer point fixes several important bugs in Hammer, as well as fixing interoperability issues that are required before an upgrade to Infernalis.  That is, all users of earlier version of Hammer or any version of Firefly will first need to upgrade to hammer v0.94.4 or later *before* upgrading to Infernalis (or future releases).

**Pull request merged**